docs/api-reference/mapbox/overview.md

```diff
@@ -30,6 +31,17 @@ npm install @deck.gl/mapbox
 import {MapboxOverlay} from '@deck.gl/mapbox';
 ```
 
+## Examples
+
+[test/apps/mapbox-integration](https://github.com/visgl/deck.gl/tree/8.9-release/test/apps/mapbox-integration)
```

Comment:
Link to master or release?

Reply:
We should not link to test/apps from docs. These are not maintained with the mindset that they will be used by external users. If you think our existing examples are insufficient you can add more to examples/get-started

Reply:
Ok, that makes sense. I'd like to maintain two public examples for the mapbox integration using interleaved MapboxOverlay with maplibre in react and pure-js since it's a common usage when someone reaches for @deck.gl/mapbox.